### PR TITLE
Reformat section 14 (classes)

### PR DESCRIPTION
**Pull request by jskeet**
(This is out of order as I'd thought classes came after expressions...)

### CODE REVIEW

**Review by RexJaeschke — APPROVED** [review]

File: standard/classes.md [hunk 1]
```diff
@@ -517,10 +528,12 @@ Values of a constrained type parameter type can be used to access the instance m
...
-> class Printer<T> where T: IPrintable
+>
+> class Printer<T> where T : IPrintable
 > {
-> void PrintOne(T x) {
->     x.Print();
+>     void PrintOne(T x)
+>     {
+>         x.Print();
 >     }
```
Comment (BillWagner) [inline standard/classes.md]: Again, I'd prefer expression bodied members:

```suggestion
>     void PrintOne(T x) => x.Print();
```

File: standard/classes.md [hunk 2]
```diff
@@ -787,7 +810,8 @@ The inherited members of a constructed class type are the members of the immedia
 > {
 >     public U F(long index) {...}
 > }
-> class D<T>: B<T[]>
+>
+> class D<T> : B<T[]>
 > {
 >     public T` G(string s) {...}
```
Comment (BillWagner) [inline standard/classes.md]: Is that a stray back-tick?

```suggestion
>     public T G(string s) {...}
```

File: standard/classes.md [hunk 3]
```diff
@@ -1432,7 +1512,9 @@ The initial value of a field, whether it be a static field or an instance field,
 > {
 >     static bool b;
 >     int i;
->     static void Main() {
+>
+>     static void Main()
+>     {
 >         Test t = new Test();
 >         Console.WriteLine("b = {0}, i = {1}", b, t.i);
```
Comment (BillWagner) [inline standard/classes.md]: string interpolation:

```suggestion
>         Console.WriteLine($"b = {b}, i = {t.i}");
```

File: standard/classes.md [hunk 4]
```diff
@@ -1453,12 +1535,15 @@ Field declarations may include *variable_initializer*s. For static fields, varia
...
 > {
 >     static double x = Math.Sqrt(2.0);
 >     int i = 100;
 >     string s = "Hello";
->     static void Main() {
+>
+>     static void Main()
+>     {
 >         Test a = new Test();
 >         Console.WriteLine("x = {0}, i = {1}, s = {2}", x, a.i, a.s);
```
Comment (BillWagner) [inline standard/classes.md]: same:
```suggestion
>         Console.WriteLine($"x = {x}, i = {a.i}, s = {a.s}");
```

File: standard/classes.md [hunk 5]
```diff
@@ -1477,11 +1562,14 @@ It is possible for static fields with variable initializers to be observed in th
...
+>
 > class Test
 > {
 >     static int a = b + 1;
 >     static int b = a + 1;
->     static void Main() {
+>
+>     static void Main()
+>     {
 >         Console.WriteLine("a = {0}, b = {1}", a, b);
```
Comment (BillWagner) [inline standard/classes.md]: same:

```suggestion
>         Console.WriteLine($"a = {a}, b = {b}");
```

File: standard/classes.md [hunk 6]
```diff
@@ -1501,18 +1589,23 @@ The static field variable initializers of a class correspond to a sequence of as
...
 > class Test
 > {
->     static void Main() {
->     Console.WriteLine("{0} {1}", B.Y, A.X);
-> }
-> public static int F(string s) {
->     Console.WriteLine(s);
+>     static void Main()
+>     {
+>         Console.WriteLine("{0} {1}", B.Y, A.X);
```
Comment (BillWagner) [inline standard/classes.md]: again:

```suggestion
>         Console.WriteLine($"{B.Y} {A.X}");
```

File: standard/classes.md [hunk 7]
```diff
@@ -1535,19 +1628,24 @@ The static field variable initializers of a class correspond to a sequence of as
 > using System;
 > class Test
 > {
->     static void Main() {
+>     static void Main()
+>     {
 >         Console.WriteLine("{0} {1}", B.Y, A.X);
```
Comment (BillWagner) [inline standard/classes.md]: same:

```suggestion
>         Console.WriteLine($"{B.Y} {A.X}");
```

File: standard/classes.md [hunk 8]
```diff
@@ -1777,16 +1875,20 @@ A method declared as an iterator ([§14.14](classes.md#1414-iterators)) may not
...
 >     }
-> static void Main() {
->        int i = 1, j = 2;
->        Swap(ref i, ref j);
+>
+>     static void Main()
+>     {
+>         int i = 1, j = 2;
+>         Swap(ref i, ref j);
 >         Console.WriteLine("i = {0}, j = {1}", i, j);
```
Comment (BillWagner) [inline standard/classes.md]: same suggestion:

```suggestion
>         Console.WriteLine($"i = {i}, j = {j}");
```

File: standard/classes.md [hunk 9]
```diff
@@ -1880,13 +1992,18 @@ Except for allowing a variable number of arguments in an invocation, a parameter
 > using System;
 > class Test
 > {
->     static void F(params int[] args) {
+>     static void F(params int[] args)
+>     {
 >         Console.Write("Array contains {0} elements:", args.Length);
```
Comment (BillWagner) [inline standard/classes.md]: same:

```suggestion
>         Console.Write($"Array contains {args.Length} elements:");
```

File: standard/classes.md [hunk 10]
```diff
@@ -1880,13 +1992,18 @@ Except for allowing a variable number of arguments in an invocation, a parameter
...
 >         foreach (int i in args)
->         Console.Write(" {0}", i);
+>         {
+>             Console.Write(" {0}", i);
```
Comment (BillWagner) [inline standard/classes.md]:
```suggestion
>             Console.Write($" {i}");
```

File: standard/classes.md [hunk 11]
```diff
@@ -2152,14 +2285,16 @@ An override declaration can access the overridden base method using a *base_acce
...
+>
+> class B : A
 > {
 >     int y;
->     public override void PrintFields() {
+>
+>     public override void PrintFields()
+>     {
 >         base.PrintFields();
 >         Console.WriteLine("y = {0}", y);
```
Comment (BillWagner) [inline standard/classes.md]:
```suggestion
>         Console.WriteLine($"y = {y}");
```

**Review by BillWagner — APPROVED** [review]
once again, this is great, and only a few nits that I noticed.

I think we can declare success on this project!  🎆

**Pull request merged**